api/post.go

```diff
@@ -300,6 +297,9 @@ func handlePostEvents(c *Context, post *model.Post, triggerWebhooks bool) {
 		channel = result.Data.(*model.Channel)
 	}
 
+	InvalidateCacheForChannel(channel)
```

Comment:
is there a reason why this is moved down?

I would guess the more time between this and publish would be better.